modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/service/test/JournalFolderLocalServiceTest.java

```diff
@@ -74,7 +150,49 @@ public void testGetNoAssetFolders() throws Exception {
 		Assert.assertEquals(folders.toString(), folder, folders.get(0));
 	}
 
+	@Test(expected = DuplicateJournalFolderExternalReferenceCodeException.class)
+	public void testJournalFolderWithExistingExternalReferenceCode()
+		throws Exception {
+
+		JournalFolder folder = addJournalFolder(
+			RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+			RandomTestUtil.randomString());
+
+		addJournalFolder(
+			folder.getExternalReferenceCode(), RandomTestUtil.randomString(),
+			folder.getDescription());
+	}
+
+	@Test
+	public void testUpdateJournalFolderByExternalReferenceCode()
```

Comment:
We are not testing anything here. I would delete this test

Reply:
+1